src/guess-the-number.js

```diff
@@ -0,0 +1,26 @@
+const readlineSync = require('readline-sync')
+const prompt = readlineSync.question
+
+const guessTheNumber = guess => {
+  const randomNumber = randomNumberGenerator()
+    if ( guess > randomNumber ) {
```

Comment:
Will do--this was leftover indentation from when these conditionals were part of another function block.